### PR TITLE
docs(region-endpoints): Some refactoring to not have duplicated regions information everywhere

### PR DESCRIPTION
**Pull request by brandon-welsch**
A "little" PR of refactoring to have endpoints documented in multiple files but at one place.
Also updated a couple of URLs and make some improvements

### CODE REVIEW

**Comment by brandon-welsch** [comment]
All pages checked, it's better now 😃

File: src/_posts/databases/mongodb/2000-01-01-compass.md [hunk 1]
```diff
@@ -41,28 +41,29 @@ Fill the fields accordingly:
 {% assign img_url = "https://cdn.scalingo.com/documentation/compass/new_connection.png" %}
 {% include mdl_img.html %}
 
-The "SSH Tunnel" section must "Use Identity File". The identity file to provide
+The `SSH Tunnel` section must `Use Identity File`. The identity file to provide
```
Comment (EtienneM) [inline src/_posts/databases/mongodb/2000-01-01-compass.md:44]: I wouldn't write "SSH Tunnel" and "Use Identity File" as a code, it's not a code at all 🤔

File: src/_posts/platform/databases/2000-01-01-access.md [hunk 2]
```diff
@@ -154,13 +157,13 @@ Example:
 If the app is in the region `osc-fr1` and if the environment variable is the following:
 
 ```
-SCALINGO_POSTGRESQL_URL=postgresql://user:secret@my-db.postgresql.dbs.appsdeck.eu:30000/my-db
+SCALINGO_POSTGRESQL_URL=postgresql://user:secret@my-db.postgresql.dbs.scalingo.com:30000/my-db
```
Comment (EtienneM) [inline src/_posts/platform/databases/2000-01-01-access.md:160]: There are other places in the doc with `appsdeck.eu`. Can you also update these URLs?

File: src/_posts/platform/internals/2000-01-01-regions.md [hunk 3]
```diff
@@ -1,48 +1,48 @@
...
 
-When using the web dashboard at
-[https://my.scalingo.com](https://my.scalingo.com), you will automatically be
-redirected to the region you have access to. If your account has access to
-multiple regions, a selection screen will be presented to let you chose which
-region you want to use:
-
-{% assign img_url = "https://cdn.scalingo.com/documentation/internals/screenshot_dashboard_regions_selection.png" %}
-{% include mdl_img.html %}
+The Web Dashboard is available at
```
Comment (EtienneM) [inline src/_posts/platform/internals/2000-01-01-regions.md:32]:
```suggestion
The web dashboard is available at
```

File: src/_posts/platform/internals/2000-01-01-regions.md [hunk 4]
```diff
@@ -1,48 +1,48 @@
...
 
 ### Using the Web Dashboard
 
-When using the web dashboard at
-[https://my.scalingo.com](https://my.scalingo.com), you will automatically be
-redirected to the region you have access to. If your account has access to
-multiple regions, a selection screen will be presented to let you chose which
-region you want to use:
-
-{% assign img_url = "https://cdn.scalingo.com/documentation/internals/screenshot_dashboard_regions_selection.png" %}
```
Comment (EtienneM) [inline src/_posts/platform/internals/2000-01-01-regions.md:32]: Can you also remove this screenshot from the CDN as it's not used anywhere else?

File: src/_posts/platform/internals/2000-01-01-regions.md [hunk 5]
```diff
@@ -1,48 +1,48 @@
...
+And when you want to create an application, you will be presented with a
+selection screen to allow you to choose the region on which you want to
+create the application.
 
 ### Using the CLI
 
-If you use the CLI to manage your app, all commands now have the flag `--region`
-to specify the region you want to use.
+If you use the CLI to manage your application, all commands now have the
+flag `--region` to specify the region you want to use.
```
Comment (EtienneM) [inline src/_posts/platform/internals/2000-01-01-regions.md:45]:
```suggestion
If you use the CLI to manage your application, all commands have the
flag `--region` to specify the region you want to use.
```

**Review by EtienneM — APPROVED** [review]
Thanks for this refactoring. Just a few small comments but LGTM :)

File: src/_posts/platform/internals/2000-01-01-regions.md (hunk 4 above)
Comment (brandon-welsch) [reply src/_posts/platform/internals/2000-01-01-regions.md:32]: It will be deleted later in the day

**Review by EtienneM — APPROVED** [review]

**Pull request merged**